mm2src/mm2_event_stream/src/streamer_ids.rs

```diff
@@ -0,0 +1,108 @@
+use serde::de::{self, Visitor};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt;
+
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub enum StreamerId {
```

Comment:
Why not make this streamer list dynamic and agnostic about existing event sources?
I think, modules, which want to send events, may register themselves as event sources in the streamer. In register call they provide event prefixes. In response the streamer would return generated streamer_id which is used as a handle to streamer functions.
With that, we may add new modules as event sources, without touching the streamer code.
@mariocynicys